linux_os/guide/services/ntp/chronyd_or_ntpd_set_maxpoll/ansible/shared.yml

```diff
@@ -34,28 +34,29 @@
     path: {{{ chrony_conf_path }}}
   register: chrony_conf_exist_result
 
-- name: Get get conf files from {{{ chrony_conf_path }}}
-  shell: |
-    set -o pipefail
-    CHRONY_NAME={{{ chrony_conf_path }}}
-    CHRONY_PATH=${CHRONY_NAME%%.*}
-    find ${CHRONY_PATH}.* -type f -name '*.conf'
-  register: update_chrony_files
-  when: chrony_conf_exist_result.stat.exists
-  changed_when: False
+- name: Set chrony_conf facts
+  set_fact:
```

Comment:
Please, use FQCN for Ansible module names: https://complianceascode.readthedocs.io/en/latest/manual/developer/04_style_guide.html#ansible